crates/rust-analyzer/src/main_loop.rs

```diff
@@ -427,8 +434,11 @@ fn loop_turn(
         });
     }
 
+    let show_progress = !loop_state.workspace_loaded
```

Comment:
Maybe move this closer to usage?

Reply:
The problem to move closer here is `loop_state.workspace_loaded` will be changed to the following code.